CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Change Log
 
+## Unreleased
+
+- Upgrade to `parse5@^v2.2.22` so that we can take advantage of new types info.
```

Comment:
Yeah, it was covered by the previous semver range so there's almost no way in which this would affect someone.